### PR TITLE
ICP-5724 : Add OCF device type to Fibaro wall plugs

### PR DESCRIPTION
**Pull request by munds**
cc @greens @workingmonk

### CODE REVIEW

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *	Fibaro Wall Plug ZW5
  */
 metadata {
-	definition (name: "Fibaro Wall Plug EU ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+	definition (name: "Fibaro Wall Plug EU ZW5", namespace: "FibarGroup", author: "Fibar Group", ocfDeviceType: "oic.d.martplug") {
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: typo. should be `smartplug`

Comment (greens) [reply devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: in all of them actually

**Review by greens — CHANGES_REQUESTED** [review]
typo in all three

**Comment by munds** [comment]
Made the requested changes, @greens

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
going to merge this for you so it makes it to staging

**Pull request merged**